src/main/python/hydra/helloworldtest/runtest.py

```diff
@@ -0,0 +1,84 @@
+import time
+
+from hydra.lib.runtestbase import RunTestBase
+try:
+    # Python 2.x
+    from ConfigParser import ConfigParser
+except ImportError:
+    # Python 3.x
+    from configparser import ConfigParser
+
+
+class HW(RunTestBase):
+    def __init__(self):
+        self.config = ConfigParser()  # Object contains 'configurations' parsed from hydra.ini config file
+        RunTestBase.__init__(self, 'HelloWorld', None, None, startappserver=True, mock=False)
+        self.hw_server_app_id = super(HW, self).format_appname("/hw-server")
+        self.hw_client_app_id = super(HW, self).format_appname("/hw-client")
+        self.hw_server_task_ip = None
+        self.hw_server_task_port = None
+        self.hw_client_task_ip = None
+        self.hw_client_task_port = None
+        self.add_appid(self.hw_server_app_id)
+        self.add_appid(self.hw_client_app_id)
+
+    def run_test(self):
+        # Get Mesos/Marathon client
+        super(HW, self).start_init()
+        # Launch HelloWorld server
+        self.launch_hw_server()
+        # Launch HelloWorld client
+        self.launch_hw_client()
+
+    def launch_hw_server(self):
+        print ("Launching the HelloWorld server app")
+        constraints = [self.app_constraints(field='hostname', operator='UNIQUE')]
```

Comment:
I added it as I wanted to run the publisher only on slave-set1_0. Why I tried to launch it only on slave-set1_0? I just wanted to showcase how one can add constraints using Hydra.
I can remove it.